nexus/src/app/instance.rs

```diff
@@ -488,11 +492,20 @@ impl super::Nexus {
             .derive_guest_network_interface_info(&opctx, &authz_instance)
             .await?;
 
-        let external_ip = self
+        // Collect the external IPs for the instance.
+        //
+        // For now, we take the Ephemeral IP, if it exists, or the SNAT IP if not.
+        // TODO-correctness: Handle multiple IP addresses
+        // TODO-correctness: Handle Floating IPs
```

Comment:
Yeah, will do. I was planning to switch the partition function used here, to collect all SNAT IPs and then everything else. That's because from OPTE's or Dendrite's point of view, Ephemeral and Floating IPs are the same. But I'll track that in the Floating IP issue, and add that check here.

Reply:
Actually, that's silly. I'm just going to switch the partition to collect SNAT and everything else _now_.